src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -108,5 +111,25 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
     const durationColumnIndex = 3;
     return column === durationColumnIndex ? moment.duration(dataFormated).asHours().toFixed(2) : dataFormated;
   }
+  
+  sumDates(arrayData: Entry[]): TotalHours{
+    this.resultSum = new TotalHours();
+    let arrayDurations= new Array();
+    arrayData.forEach(entry =>{
+      let start = moment(entry.end_date).diff(moment(entry.start_date));
+      arrayDurations.push(moment.utc(start).format("HH:mm:ss"));
+    });
+    
+    let totalDurations = arrayDurations.slice(1)
+    .reduce((prev, cur) => {
+      return prev.add(cur);
+    },
+    moment.duration(arrayDurations[0]));
+    let daysInHours = totalDurations.days() >= 1 ? totalDurations.days() * 24 :  0;
```

Comment:
@sbateca , can you explain to me how the resultSum is calculated? I guess the condition you are doing here is not neccesary, but first help me explaining to me what is going on with the entire calculation please. Thanks pana..!!

Reply:
@almeida-erick, the total of the durations is assigned in a moment.Duration object. Accord  to the purpose of this ticket is necessary shows the total time in hours. For this reason, the days are converted in hours.